File: Lab1/Cau_3/app/src/main/java/com/example/myapplication/ProfileActivity.java
package com.example.myapplication;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

public class ProfileActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_profile);

        // Ẩn thanh hành động
        if (getSupportActionBar() != null) {
            getSupportActionBar().hide();
        }

        // Khởi tạo các view
        CardView backButton = findViewById(R.id.backButton);
        CardView editButton = findViewById(R.id.editButton);
        ImageView profileImage = findViewById(R.id.profileImage);
        TextView tvName = findViewById(R.id.tvName);
        TextView tvLocation = findViewById(R.id.tvLocation);

        // Xử lý sự kiện khi nhấn nút quay lại
        backButton.setOnClickListener(v -> {
            finish(); // Đóng activity hiện tại
        });

        // Xử lý sự kiện khi nhấn nút chỉnh sửa
        editButton.setOnClickListener(v -> {
            Toast.makeText(this, "Chỉnh sửa hồ sơ", Toast.LENGTH_SHORT).show();
            // Code mở màn hình chỉnh sửa
        });
    }
}

File: Lab1/Cau_3/app/src/main/java/com/example/myapplication/ProfileActivity.kt
package com.example.myapplication

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView
import com.bumptech.glide.Glide

class ProfileActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_profile)

        // Load ảnh từ URL vào profileImage
        val imageUrl = "https://hoanghamobile.com/tin-tuc/wp-content/uploads/2024/06/anh-itachi-2.jpg"
        val profileImage = findViewById<android.widget.ImageView>(R.id.profileImage)
        Glide.with(this)
            .load(imageUrl)
            .placeholder(R.drawable.profile_placeholder)
            .error(R.drawable.profile_placeholder)
            .circleCrop()
            .into(profileImage)

        // Xử lý nút back
        findViewById<CardView>(R.id.backButton).setOnClickListener {
            finish()
        }

        // Xử lý nút email validation
        findViewById<CardView>(R.id.emailValidationButton).setOnClickListener {
            val intent = Intent(this, EmailValidationActivity::class.java)
            startActivity(intent)
        }
    }
} 